service/static/js/rest_api.js

```diff
@@ -30,7 +27,7 @@ $(function () {
         $("#customer_last_name").val("");
         $("#customer_email").val("");
         $("#customer_password").val("");
-        $("#customer_status").val(STATUS.ACTIVE);
+        $("#customer_status").val("STATUS.ACTIVE");
```

Comment:
No need to put this enum in quotes. Putting this in quotes unintentionally sets customer status to the string `"STATUS.ACTIVE"` which is not an accepted value for that field